vb-fe/src/App.tsx

```diff
@@ -47,26 +44,29 @@ export const App = (): JSX.Element => {
   };
 
   const AuthenticatedApp = (): JSX.Element => {
-    const { connect } = useSocket();
-
     useEffect(() => {
-      connect();
       initHttp();
-    }, [connect]);
+      initPipeline();
```

Comment:
Are you sure we have to do this? Have you tested things w/out it?

Reply:
Do what? This is initializing the sockets, and the publishers/subscribers